modules/markup/markdown/markdown.go

```diff
@@ -104,22 +114,14 @@ func (r *Renderer) Image(out *bytes.Buffer, link []byte, title []byte, alt []byt
 		prefix = util.URLJoin(prefix, "wiki", "raw")
 	}
 	prefix = strings.Replace(prefix, "/src/", "/raw/", 1)
-	if len(link) > 0 {
-		if markup.IsLink(link) {
-			// External link with .svg suffix usually means CI status.
-			// TODO: define a keyword to allow non-svg images render as external link.
-			if bytes.HasSuffix(link, svgSuffix) || bytes.Contains(link, svgSuffixWithMark) {
```

Comment:
@LER0ever is it ok that the SVG case is no longer being handled?

Reply:
@techknowlogick to be honest I don't see any reason to differentiate svg files with oter filetypes. They could all be ci badges or normal images. So I just treated all the images the same.  Do you have any specific case where svgs needs to be handled separatedly?

Reply:
@lafriks right, done through faade68